src/config.ts

```diff
@@ -83,4 +85,6 @@ function reporter<T>({ errors }: envalid.ReporterOptions<T>) {
   })
 }
 
-export const getConfig = memoize(() => cleanEnv(env, validators, { reporter }))
+export const getConfig = memoize(() =>
```

Comment:
how do you feel about not merging here, and just exposing a `constants` object such that consumers aren't confused about reactivity?

Reply:
Reduce complexity? What a reasonable suggestion.

Actioned in https://github.com/shapeshift/web/pull/2209/commits/473d82888bad788364cc5c90b373b337389fdb11